bellows/cli/util.py

```diff
@@ -112,9 +113,9 @@ def cfg(config_id, value):
 
 
 @asyncio.coroutine
-def setup_application(dev, database_file, startup=True):
+def setup_application(dev, database_file, startup=True, baudrate=None):
```

Comment:
Make it required here, too (and next to dev).

Reply:
I will make it required.